services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+
+from django.db.models.query import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: datetime.datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    movie_session = MovieSession.objects.create(show_time=movie_show_time,
+                                                movie_id=movie_id,
+                                                cinema_hall_id=cinema_hall_id)
+    return movie_session
+
+
+def get_movies_sessions(session_date: datetime = None) -> QuerySet:
+    movie_session = MovieSession.objects.all()
+
+    if session_date:
+        movie_session = MovieSession.objects.filter(
+            show_time__date=session_date
+        )
+
+    return movie_session
+
+
+def get_movie_session_by_id(movie_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_id)
+
+
+def update_movie_session(session_id: int,
+                         show_time: datetime = None,
+                         movie_id: int = None,
+                         cinema_hall_id: int = None) -> MovieSession:
+    movie_session = MovieSession.objects.get(id=session_id)
+    if show_time:
+        movie_session.show_time = show_time
+    if cinema_hall_id:
+        movie_session.cinema_hall_id = cinema_hall_id
+    if movie_id:
+        movie_session.movie_id = movie_id
+    movie_session.save()
+    return movie_session
+
+
+def delete_movie_session_by_id(session_id: int) -> QuerySet | None:
+    movie_session = MovieSession.objects.get(id=session_id)
```

Comment:
you can do this in one line.
and no need to return a deleted instance in the end